sql/v1.sql

```diff
@@ -0,0 +1,26 @@
+create table if not exists theme (
```

Comment:
Rename this file explaining what it is doing: `v1_create_theme_table` for instance